ReleaseHistory.md

```diff
@@ -14,7 +14,6 @@
 * BRK: Remove unused `quiet` parameter from `SarifLogger`. [#2639]https://github.com/microsoft/sarif-sdk/pull/2639
 * BRK: Remove `ComputeHashData` and `AnalysisTargetToHashDataMap` properties from `SarifLogger` (in preference of new `fileRegionsCache` parameter. [#2639](https://github.com/microsoft/sarif-sdk/pull/2639)
 * BRK: Eliminate proactive hashing of artifacts in `SarifLogger` constructor when `OptionallyEmittedData.Hashes` is specified. [#2639](https://github.com/microsoft/sarif-sdk/pull/2639)
-* BUG: Fixed incorrect hashes being returned when `--insert hashes` is enabled. [#2667](https://github.com/microsoft/sarif-sdk/pull/2667)
```

Comment:
>BUG: Fixed incorrect hashes being returned when `--insert hashes` is enabled. [](http://example.com/codeflow?start=2&length=77)

since the revert happens within one unreleased version,
I did not have 2 entries and removed it,
let me know if we like to have 2 entries instead I will update it.